### PR TITLE
[CI-5969]: reduce log level to debug on some info

### PR DESCRIPTION
**Pull request by jamie-harness**
Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on our [code of conduct](CODE_OF_CONDUCT.md), and the process for submitting pull requests to us.

Fixes #

## Proposed Changes

-
-
-

### Checklist

- [ ] Read the [**CONTRIBUTING**](CONTRIBUTING.md) document.
- [ ] Read the [**CODE OF CONDUCT**](CODE_OF_CONDUCT.md) document.
- [ ] Add tests to cover changes.
- [ ] Ensure your code follows the code style of this project.
- [ ] Ensure CI and all other PR checks are green OR
    - [ ] Code compiles correctly.
    - [ ] Created tests which fail without the change (if possible).
    - [ ] All new and existing tests passed.
- [ ] Add your changes to `Unreleased` section of [CHANGELOG](CHANGELOG.md).
- [ ] Improve and update the [README](README.md) (if necessary).
- [ ] Ensure [documentation](./DOCS.md) is up-to-date. The same file will be updated in [plugin index](https://github.com/drone/drone-plugin-index/blob/master/content/meltwater/drone-cache/index.md) when your PR is accepted, so it will be available for end-users at http://plugins.drone.io.

### CODE REVIEW

File: archive/tar/tar.go [hunk 1]
```diff
@@ -209,7 +209,7 @@ func (a *Archive) Extract(dst string, r io.Reader) (int64, error) {
 			target = filepath.Join(dst, name)
 		}
 
-		level.Info(a.logger).Log("msg", "extracting archive", "path", target)
```
Comment (hemanthmantri) [inline archive/tar/tar.go:212]: If we logging this as part of restore cache step and this is one line for the tar, we should keep it.

**Review by hemanthmantri — APPROVED** [review]

File: archive/tar/tar.go (hunk 1 above)
Comment (jamie-harness) [reply archive/tar/tar.go:212]: This is the most problematic one. It prints one per file

Comment (hemanthmantri) [reply archive/tar/tar.go:212]: I see. Let's print one time to indicate we are extracting the archive to the destination (if we are not doing that already).

Comment (jamie-harness) [reply archive/tar/tar.go:212]: we have it at the beginning and ending

**Comment by shubham149** [comment]
Build is failing for it

**Comment by jamie-harness** [comment]
> Details

@shubham149 can you help re-trigger? I don't see why it failed. https://cloud.drone.io/drone-plugins/drone-meltwater-cache/141/3/1

**Comment by liangyuanpeng** [comment]
CI is all of green and the status is failed :(

**Comment by shubham149** [comment]
Linux arm build is not triggered. Hence build failed. Closing and reopening PR won't re-trigger the checks.

File: internal/plugin/auto_detect/prepare_gradle.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+)
+
+type gradlePreparer struct {}
+
+func newGradlePreparer() *gradlePreparer {
+	return &gradlePreparer{}
+}
+func (*gradlePreparer) PrepareRepo() error {
+	fileName := "gradle.properties"
+	pathToCache := filepath.Join("harness", "harnesscache")
```
Comment (shubham149) [inline internal/plugin/auto_detect/prepare_gradle.go:17]: We cannot harness related keywords in this repo. It is a drone plugin that is used by drone customers as well.

Comment (jamie-harness) [reply internal/plugin/auto_detect/prepare_gradle.go:17]: oh sorry this is suppose to be a different PR

**Review by tphoney — APPROVED** [review]

**Comment by tphoney** [comment]
closing this based on #23

**Pull request closed without merging**